src/features/newTab/default/settings/index.tsx

```diff
@@ -0,0 +1,55 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+import { StyledSettingsMenu, StyledSettingsRow, StyledSettingsText, StyledSettingsTitle } from './style'
+
+import { Toggle } from '../../../shields/toggle'
+
+interface Props {
+  onClickOutside: () => void
+  toggleShowBackgroundImage: () => void
+  showBackgroundImage: boolean
+  theme: 'lightMode' | 'darkMode'
```

Comment:
We can't use a prop named 'theme' since we use styled-component's theme provider which adds `theme` property to all styled-components props. Instead simply add or use a color property in the ITheme (e.g. `props.theme.colors.contextMenuBackground`) or, less preferably, use `props.theme.name` to get 'dark' or 'light'. See shields as example.